hibernate-core/src/main/java/org/hibernate/event/internal/DefaultSaveOrUpdateEventListener.java

```diff
@@ -99,8 +99,7 @@ protected Serializable performSaveOrUpdate(SaveOrUpdateEvent event) {
 	}
 
 	protected Serializable entityIsPersistent(SaveOrUpdateEvent event) throws HibernateException {
-		final boolean traceEnabled = LOG.isTraceEnabled();
```

Comment:
I understand why we did it when there is a static field. I'm curious though on this pattern here?

If it doesn't save us anything, I'm all for consistency but it seems to be a bit different so I thought I might as well ask.

Reply:
I made those changes for consistency. I searched through the code base for all occurrences where we assign to a variable as the call gets inlined there didn't seem much sense to store in a variable. I can revert those changes if needed

Reply:
I made those changes for consistency. I searched through the code base for all occurrences where we assign to a variable as the call gets inlined there didn't seem much sense to store in a variable. I can revert those changes if needed

Reply:
I made those changes for consistency. I searched through the code base for all occurrences where we assign to a variable as the call gets inlined there didn't seem much sense to store in a variable. I can revert those changes if needed